### PR TITLE
Devdocs: document the process for making a new release of this action

### PR DESCRIPTION
**Pull request by DilumAluthge**

### CODE REVIEW

File: devdocs/making_a_new_release.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+# Create a new v2 tag locally, where the new v2 tag will point to the
+# release that you created in the previous step.
+#
+# Make sure to change `v2.2.0` to the actual value for the release
+# that you just created in the previous step.
+git tag v2 v2.2.0
+
+# Force-push the new v2 tag:
+git push --force origin v2
```
Comment (omus) [inline devdocs/making_a_new_release.md:24]:
```suggestion
git fetch --all --tags

# Create/update the new v2 tag locally, where the new v2 tag will point to the
# release that you created in the previous step.
#
# Make sure to change `v2.2.0` to the actual value for the release
# that you just created in the previous step.
git tag -f v2 v2.2.0

# Force-push the new v2 tag:
git push origin tag v2 --force
```

**Review by omus — APPROVED** [review]
Looks good. I streamlined the instructions

File: devdocs/making_a_new_release.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+```bash
+git clone git@github.com:julia-actions/cache.git
+cd cache
+git fetch --all --tags
+
+# Create/update the new v2 tag locally, where the new v2 tag will point to the
+# release that you created in the previous step.
+#
+# Make sure to change `v2.2.0` to the actual value for the release
+# that you just created in the previous step.
```
Comment (DilumAluthge) [inline devdocs/making_a_new_release.md:16]:
```suggestion
# that you just created in the previous step.
#
# The `-f` flag forcible overwrites the old
# `v2` tag (if it exists).
```

File: devdocs/making_a_new_release.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+#
+# The `-f` flag forcible overwrites the old
```
Comment (DilumAluthge) [inline devdocs/making_a_new_release.md:18]:
```suggestion
# The `-f` flag forcibly overwrites the old
```

**Comment by DilumAluthge** [comment]
@omus @IanButterworth Can you take another look? I made one more small tweak.

**Comment by IanButterworth** [comment]
This seems to be documenting the standard release method in a backward way?

I git tag, then push tags, then name a release from the new tag?

**Comment by DilumAluthge** [comment]
Sure, I can edit the docs to reflect that workflow.

@IanButterworth When you create the new tag (e.g. the `v2.2.0` tag) locally, do you create a lightweight tag or an annotated tag?

That is, do you run `git tag v2.2.0` (lightweight tag) or `git tag -a v2.2.0` (annotated tag)?

**Comment by IanButterworth** [comment]
`git tag v2.2.0 commit_hash`

**Comment by DilumAluthge** [comment]
Alright @IanButterworth take a look now and let me know what you think.

**Review by IanButterworth — APPROVED** [review]

**Pull request merged**